config/gflownet/gflownet.yaml

```diff
@@ -80,3 +49,12 @@ policy:
     reload_ckpt: False
   backward: null
   ckpt_period: null
+num_empirical_loss: 200000
+oracle:
+    # Number of samples for oracle metrics
+    n: 500
+# remove eventually
```

Comment:
note: the following parameters will be removed once we finalize how we'd like to implement these modes